src/pocketmine/resources/creativeitems.json

```diff
@@ -593,6 +593,24 @@
         "count": 1,
         "nbt": ""
     },
+	{
```

Comment:
This file is auto-generated, don't modify it by hand. It should already be up-to-date for 0.16.